parking_permits/talpa/order.py

```diff
@@ -70,7 +126,34 @@ def _create_customer_data(cls, customer):
     @classmethod
     def _create_order_data(cls, order):
         order_items = order.order_items.all().select_related("product", "permit")
-        items = [cls._create_item_data(order, order_item) for order_item in order_items]
+        permits = set([item.permit for item in order_items])
+        grouped_items_by_permit = [
+            list(
+                filter(
+                    lambda item: item.permit == permit and item.quantity, order_items
+                )
+            )
+            for permit in permits
+        ]
+        items = []
```

Comment:
IMO, these lines are a bit too complex. Using a `defaultdict` will simplify the code:

```python
order_items_by_permit = defaultdict(list)
for order_item in order_items:
    order_items_by_permit[order_item.permit].append(order_item)
```

Reply:
Done